packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -245,8 +256,10 @@ def remove_suffix(string: str) -> str:
     return string
 
 
-def is_python_identifier(input_string: str) -> bool:
+def is_python_identifier(input_string: Optional[str]) -> bool:
```

Comment:
hmmm, out of scope for this PR because this is already here.

@dixonjoel, consider having this renamed to is_valid_python_identifier
also, why is this method public??